### DISCUSSION TITLE
Pricing Model for test purposes only

### DISCUSSION THREAD

**Discussion by isyedakhtar** [kirill-zhirnov/boundless-nextjs-sample#88] [Q&A]
I can see a free trial available for non-commercial usage. This free trial has limit of 50MB data. Can someone tell, what does this 50MB mean? What data we are talking about here. I am only using the OOTB test products and categories. Does this data reflect the orders that will be places etc? How can I check my quota with the passage of time.

**Comment by kirill-zhirnov** [comment]
Hi @isyedakhtar 
Thank you for your question!

50MB is a space for your assets: files, images, etc. It doesn't include DB size. You can see actually consuming space at Settings -> My Account.

**Reply by kirill-zhirnov** [reply]
P.S. I would recommend to clean the Database before usage, so you have no demo-data.
